src/App.js

```diff
@@ -1,13 +1,20 @@
-import React from "react";
+import React, { Route } from "react";
+import Form from "./components/Form";
 
 const App = () => {
 	return (
 		<>
 			<header>
 				<h1>Lambda Eats</h1>
-				<Route></Route>
-				<Route></Route>
+				<button>Home</button>
+				<button>Help</button>
 			</header>
+
+			<div>
+				<Route path="/" component={Form}>
```

Comment:
One of the newer features of react router allows for using a child component, which also helps in passing props
`<Route path='/'><Form></Route>

It might be something to look into